libyul/backends/wasm/polyfill/Memory.yul

```diff
@@ -66,4 +66,39 @@ function memset(ptr:i32, value:i32, length:i32) {
 	{
 		i32.store8(i32.add(ptr, i), value)
 	}
-}
\ No newline at end of file
+}
+
+function mload_internal(pos:i32) -> z1, z2, z3, z4 {
+	z1 := bswap64(i64.load(pos))
+	z2 := bswap64(i64.load(i32.add(pos, 8:i32)))
+	z3 := bswap64(i64.load(i32.add(pos, 16:i32)))
+	z4 := bswap64(i64.load(i32.add(pos, 24:i32)))
+}
+
+function mstore_address(pos:i32, a1, a2, a3, a4) {
+	a1, a2, a3 := u256_to_address(a1, a2, a3, a4)
+	i64.store(pos, 0:i64)
+	i32.store(i32.add(pos, 8:i32), 0:i32)
+	i32.store(i32.add(pos, 12:i32), bswap32(i32.wrap_i64(a1)))
+	i64.store(i32.add(pos, 16:i32), bswap64(a2))
+	i64.store(i32.add(pos, 24:i32), bswap64(a3))
+}
+
+function mload_address(pos:i32) -> z2, z3, z4 {
```

Comment:
Can you document these helpers?

If I read the callsites correctly, this should read as:
```suggestion
// Reads 256-bits from `pos`, but only returns the bottom 160-bits.
function mload_address(pos:i32) -> z2, z3, z4 {
```